### ISSUE TITLE
Extended Settings causes crash on startup sometimes

### ISSUE THREAD

**Issue by Scrxtchy** [Bluscream/pyTSon_plugins#44]
Using it to remove some yuge graphics.
2/3 times when opening ts3, the plugin will crash the client.

Maybe this is caused by autojoining multiple servers, one of which takes 3+ seconds to connect to. 
I don't qt so you're guess is much better than mine

**Comment by Bluscream** [comment]
Thats a known bug and i gave up using the script because of that, there is just no way to debug it :(

**Comment by Scrxtchy** [comment]
lol

**Comment by Bluscream** [comment]
Probably because the banner is being removed for all tabs and it was removed on some already so it tries to remove smth that's already removed

**Comment by Bluscream** [comment]
Gonna take another look

**Comment by Scrxtchy** [comment]
thanks, I guess I should also note that ts3 will crash when resuming from sleep mode
This sounds consistent since the banner was already removed, if the script is processing again upon wake, it sounds like this could be a cause

**Comment by Bluscream** [comment]
Thanks for the tip

**Comment by Scrxtchy** [comment]
Hmm, would it work to make it still exist, but invisible? Like 0height?

**Comment by Bluscream** [comment]
Yeah, every qt widget has css-like style properties; which also brings me to the idea to outsource that to a theme :thinking:

**Comment by Scrxtchy** [comment]
Try banner.visible = False

**Comment by Bluscream** [comment]
Yeah, i'll rework that completely. Have you seen my recent work on https://github.com/Bluscream/pyTSon_plugins/blob/master/scripts/treeView/__init__.py

**Comment by Scrxtchy** [comment]
keyword system a fuk

**Comment by Scrxtchy** [comment]
Not perfect https://github.com/Scrxtchy/ts3-banner-hider

**Comment by Bluscream** [comment]
```css
Banner {
  display: none;
  visibility: hidden;
  height: 0px;
  width: 0px;
}
```
i thought of something like this
